client/services/FeedbackGeneratorService.js

```diff
@@ -415,6 +415,20 @@ tie.factory('FeedbackGeneratorService', [
             'Please keep your code within the existing predefined functions',
             '-- we cannot process code in the global scope.'
           ].join(' '));
+        } else if (prereqCheckFailure.hasWrongLanguage()) {
+          WRONG_LANGUAGE_ERRORS.python.forEach(function(error) {
+            if (error.errorName === prereqCheckFailure.getWrongLangKey()) {
+              error.feedbackParagraphs.forEach(function(json) {
+                if (json._type === 'text') {
```

Comment:
I didn't really think about it much when I was writing this, but would it just be better to write everything as public properties? these aren't supposed to be fleshed out objects per se but just a simplified representation of a FeedbackParagraph object.